cogs/tags.py

```diff
@@ -207,10 +221,17 @@ async def edit(self, ctx, name, *, content):
     @tag.command(aliases=("fe",))
     @checks.is_moderator()
     async def forceedit(self, ctx, name, *, content):
-        """Edits a tag by force.
+        """Edits a tag by force. Attach images/gifs/videos to automatically append their urls to the tag.
 
         You must have the Moderator role to use this."""
 
+        for attachment in ctx.message.attachments:
```

Comment:
this is repeated so maybe pull this into a function with_attachment_urls(ctx, content)

Reply:
Done! Made some modifications, is this okay?

Reply:
Will modify the image/video check once the other conversation is resolved